examples/plot_pvlaues.py

```diff
@@ -0,0 +1,195 @@
+# use epochs metadata to create design matrix for linear regression analyses
+
+# add intercept
+design = limo_epochs['2'].metadata.copy().assign(intercept=1)
+# effect code contrast for categorical variable (i.e., condition a vs. b)
+design['face a - face b'] = np.where(design['face'] == 'A', 1, -1)
+# create design matrix with named predictors
+predictors = ['intercept', 'face a - face b', 'phase-coherence']
+design = design[predictors]
+
+###############################################################################
+# --- run linear regression analysis using scikit-learn ---
+# data to be analysed
+data = limo_epochs['2'].get_data(picks_eeg)
+
+# number of epochs in data set
+n_epochs = data.shape[0]
+
+# number of channels and number of time points in each epoch
+# we'll use this information later to bring the results of the
+# the linear regression algorithm into an eeg-like format
+# (i.e., channels x times points)
+n_channels = picks_eeg.shape[0]
+n_times = len(limo_epochs['2'].times)
+
+# number of trials and number of predictors
+n_trials, n_predictors = design.shape
+# degrees of freedom
+dfs = float(n_trials - n_predictors)
+
+# vectorize (channel) data for linear regression
+Y = Vectorizer().fit_transform(data)
+
+# set up model and fit linear model
+linear_model = LinearRegression(fit_intercept=False)
+linear_model.fit(design, Y)
+
+###############################################################################
+# extract the coefficients for linear model estimator
+betas = get_coef(linear_model, 'coef_')
+
+# compute predictions
+predictions = linear_model.predict(design)
+
+# compute sum of squared residuals and mean squared error
+residuals = linear_model.predict(design) - Y
+# sum of squared residuals
+ssr = np.sum(residuals ** 2, axis=0)
+# mean squared error
+sqrt_mse = np.sqrt(ssr / dfs)
+
+# raw error terms from each predictor in design matrix:
+# here, we take the inverse of a design) matrix projections
+# (i.e., A^T*A)^-1 and extract the square root of the diagonal values.
+error_terms = \
+    np.sqrt(
+        np.diag(
+            np.linalg.pinv(
+                np.dot(design.T, design)
+            )
+        )
+    )
+
+###############################################################################
+# define dictionaries for results
+lm_betas, stderrs, t_vals, p_vals, s_vals = (dict() for _ in range(5))
+
+# define point asymptotic to zero to use as zero
+tiny = np.finfo(np.float64).tiny
+
+# loop through predictors to extract parameters
+for ind, predictor in enumerate(predictors):
+
+    # extract coefficients for predictor in question
+    beta = betas[:, ind]
+    # compute standard errors
+    stderr = sqrt_mse * error_terms[ind]
+
+    # compute t values
+    t_val = beta / stderr
+    # compute cumulative empirical distribution for t-values
+    cdf = stats.t.cdf(np.abs(t_val), dfs)
+    # and compute p-values
+    p_val = 2. * (1. - cdf)
```

Comment:
I  suspect there is a bug somewhere above here. The p-values just get way too small.

Reply:
I replaced:
```python
# compute cumulative empirical distribution for t-values
cdf = stats.t.cdf(np.abs(t_val), dfs)
# and compute p-values
p_val = 2. * (1. - cdf)
 ```
with:
```python
# and compute p-values
p_val = 2 * stats.t.sf(np.abs(t_val), dfs)
 ```
the survival function should be equivalent to the `1. - cdf` above. Although according to the scipy documentation using `stats.t.sf` should be more accurate.
> [Survival function (also defined as 1 - cdf, but sf is sometimes more accurate)](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.t.html).

Here are the results for the electrode `B8`:
![pvalues](https://user-images.githubusercontent.com/12409129/60827300-2e9f0400-a1b0-11e9-8367-11115a19309a.png)